includes/page_events.js

```diff
@@ -1,6 +1,13 @@
 const segmentCommon = require("./common");
 
 module.exports = (params) => {
+
+  const CUSTOM_PAGE_FIELDS_OBJ = {};
```

Comment:
These should be customePageFieldsObject - I'd only use upper case for constants that absolutely don't change, not for variables in methods